app/scripts/modules/ecs/src/serverGroup/configure/wizard/container/Container.tsx

```diff
@@ -24,6 +24,7 @@ interface IContainerState {
   dockerImages: IEcsDockerImage[];
   targetGroupsAvailable: string[];
   targetGroupMappings: IEcsTargetGroupMapping[];
+  dirtyTargetGroups: string [];
```

Comment:
This state is not something that is being changed in this component. You shouldn't be storing it in the state as a convenience factor to retrieve/dismiss it.

Reply:
Considering the current implementation where in the error will disappear only when user will choose the active target group from the dropdown we don't need this param on the state, so removed in the latest commit.